vms/platformvm/txs/executor/advance_time_test.go

```diff
@@ -83,12 +89,12 @@ func TestAdvanceTimeTxUpdatePrimaryNetworkStakers(t *testing.T) {
 // Ensure semantic verification fails when proposed timestamp is at or before current timestamp
 func TestAdvanceTimeTxTimestampTooEarly(t *testing.T) {
 	require := require.New(t)
-	env := newEnvironment(t, false /*=postBanff*/, false /*=postCortina*/)
+	env := newEnvironment(t, false /*=postBanff*/, false /*=postCortina*/, false /*postDurango*/)
```

Comment:
I totally agree. The top PR of [this thread](https://github.com/ava-labs/avalanchego/issues/2453) fixes the fork handling.
It replaces the boolean with an enum and introduces a latestFork enum.
I believe, for maintanability purposes, we should mark what tests are supposed to work in the latestForks and what are specific to some forks, so that we could easily switch all of them at once, whenever a new fork is introduced.